example/CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ if (BUILD_READERS)
 
     target_link_libraries (ZXingReader ZXing::ZXing stb::stb)
 
+    if (MSVC)
+        target_compile_options(ZXingReader PRIVATE -D_CRT_SECURE_NO_WARNINGS)
+    endif()
+
```

Comment:
Instead of duplicating that option for every executable, I'd suggest to add it at the top-level `CMakeListst.txt` line 22.